### ISSUE TITLE
Missing files on SonarCloud with wrong license headers

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [SonarSource/sonar-dotnet#4048]
As @MaartenGDev noticed in #4046, we have inconsistent license header in at least one file https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpDiagnosticAnalyzerContextHelper.cs
The email address is in unexpected form.

First problem: This should raise S1451 and it doesn't in SonarLint nor SC.
Second problem: The file is not present in [SonarCloud](https://sonarcloud.io/code?id=sonaranalyzer-dotnet&selected=sonaranalyzer-dotnet%3Aanalyzers%2Fsrc%2FSonarAnalyzer.CSharp%2FHelpers) at all.

**Comment by pavel-mikula-sonarsource** [comment]
Same SC issue seems to be for `analyzers\src\SonarAnalyzer.Common\Helpers\DiagnosticAnalyzerContextHelper.cs` and `analyzers\src\SonarAnalyzer.VisualBasic\Helpers\VisualBasicDiagnosticAnalyzerContextHelper.cs `

**Comment by pavel-mikula-sonarsource** [comment]
Root cause: 
```
Skipping auto generated file: src/SonarAnalyzer.VisualBasic/Helpers/VisualBasicDiagnosticAnalyzerContextHelper.cs
Skipping auto generated file: src/SonarAnalyzer.Common/Helpers/DiagnosticAnalyzerContextHelper.cs
Skipping auto generated file: src/SonarAnalyzer.CSharp/Helpers/CSharpDiagnosticAnalyzerContextHelper.cs
```

**Comment by pavel-mikula-sonarsource** [comment]
FP for noncomment travia:
`#region Register*ActionInNonGenerated`
